sample/ApiHubTrigger-Batch/function.json

```diff
@@ -0,0 +1,17 @@
+{
+  "bindings": [
+    {
+      "type": "apiHubTrigger",
+      "connectionString": "%dropbox%",
```

Comment:
@mathewc Connection property in BindingMetadata does not have a [AllowNameResolution]. how is it going to resolve the app setting? I can seem to find any sample to see how this is supposed to work?